.travis.yml

```diff
@@ -8,10 +8,22 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  # GAS commands for SDL scanning
+  - (cd arm && gas -fmt=json -out=results.json ./... > /dev/null)
```

Comment:
In some cases folks have dozens of findings and fixing them is time consuming and risky, so as a rule I'm not pushing everyone to solve things today. In this case as I said there's just one finding, so if we want to fix it or mark it as a false positive that's fine. The reason this is broken out per-folder is because the build process pulls in 3rd party code under the 'vendor' folder, and this code has additional findings that we're not triaging at this time.
Regarding pipe to tee, yes you can do this. Gas prints out some additional garbage when scanning some source files (notably packer-azure) but it doesn't do this for the go-sdk, and nothing in the current travis config is filtering this out anyway.